lib/utapi/utilities.js

```diff
@@ -206,8 +223,40 @@ function listMetrics(metricType) {
  * @return {function} - `utapi.pushMetric`
  */
 function pushMetric(action, log, metricObj) {
-    const { bucket, keys, byteLength, newByteLength,
-            oldByteLength, numberOfObjects, authInfo, canonicalID } = metricObj;
+    const {
+        bucket,
+        keys,
+        versionId,
+        byteLength,
+        newByteLength,
+        oldByteLength,
+        numberOfObjects,
+        authInfo,
+        canonicalID,
+        location,
+    } = metricObj;
+
+    if (utapiVersion === 2) {
+        const utapiObj = {
+            operationId: action,
+            timestamp:
+            bucket,
+            location,
+            objectDelta: numberOfObjects,
+            sizeDelta: byteLength,
+            incomingBytes: -byteLength || newByteLength,
```

Comment:
I'm not sure if these are right. `incomingBytes` and `outgoingBytes` represents the amount of data received or sent by the server during the operation.

Reply:
I meant to comment and not approve, but I can't figure out how to change it

Reply:
You can just re-review and it overrides your previous.

Reply:
oh so they're not a running total? would the incomingBytes on a delete operation just be zero then?

Reply:
yes & yes, the event reflects how an op would change the totals